### ISSUE TITLE
preinstall script affecting consuming projects

### ISSUE THREAD

**Issue by tbrannam** [brillout/react-streaming#37]
When updating to the latest release it has been noticed that our projects are being affected by preinstall script recently added to react-streaming.

Was it the intent to dictate the package-manager of consuming projects?  

```
npm notice 
npm ERR! code 254
npm ERR! path /app/node_modules/react-streaming
npm ERR! command failed
npm ERR! command sh -c npx only-allow pnpm
```

**Comment by brillout** [comment]
Didn’t know that, up for a pr to remove that?

**Comment by tbrannam** [comment]
Here's something that I think tries to continue the check for development, but no-ops when resolving as a dependent project. 

https://github.com/brillout/react-streaming/pull/38

**Comment by brillout** [comment]
Released in `0.3.28`.
